src/Form/RetryJob.php

```diff
@@ -52,22 +52,22 @@ class RetryJob extends ConfirmFormBase {
    *
    * @var \Drupal\os2forms_failed_jobs\Helper\Helper
    */
-  protected Helper $failedJobsHelper;
+  protected Helper $helper;
 
   /**
    * Retry job constructor.
    */
-  public function __construct(Connection $database, EntityTypeManager $entityTypeManager, Helper $failedJobsHelper) {
+  public function __construct(Connection $database, EntityTypeManager $entityTypeManager, Helper $helper) {
     $this->database = $database;
     $this->entityTypeManager = $entityTypeManager;
-    $this->failedJobsHelper = $failedJobsHelper;
+    $this->helper = $helper;
   }
 
   /**
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container): RetryJob {
-    return new static(
+    return new RetryJob(
```

Comment:
Why do you change `static` to `RetryJob` here? It works the same, but `new static` is the way it's done in most (all?) cases.

Reply:
Changed